runA11yTests.js

```diff
@@ -17,6 +17,34 @@ function resetReportDir( config ) {
 	fs.mkdirSync( config.reportDir, { recursive: true } );
 }
 
+// FIXME: Temporary pa11y workaround for color
+// contrast errors in two groups – first valid, second
+// invalid. Not suitable for production.
+// Switch to axe-core for better accuracy.
+function extractColorContrastInstances( data ) {
+	const colorContrastInstances = [];
+	let addingToArray = false;
+
+	for ( let i = 0; i < data.issues.length; i++ ) {
+		const issue = data.issues[i];
+
+		if ( issue.code === 'color-contrast' ) {
+			if ( !addingToArray ) {
+				// Start adding to the array
+				addingToArray = true;
+			}
+			// Add the current issue to the array
+			colorContrastInstances.push( issue );
+		} else if ( addingToArray && i > 0 && data.issues[i].code !== 'color-contrast' && data.issues[i - 1].code === 'color-contrast' ) {
+			// Break the loop if a different code value is encountered after adding to the array
```

Comment:
[Question] Why would we do this? If I'm reading correctly if data.issues looks like:

```
[ { code: 'color-contrast' }, { code: 'other' }, { code: 'color-contrast' } ]
```

then colorContrastInstances will only contain the first color contrast issue? Could you elaborate on why we skip the latter ones? it doesn't seem to match the function name.

Reply:
@jdlrobson I found that the first group of color contrast issues are the only ones that are valid. I discovered this while comparing the .csv list of color contrast errors to the list generated by axe-core on the browser.

Typical pattern of pa11y test output:
```
issues [ 
{code: 'color-contrast'}, // <-start: Initial set of color contrast errors - valid error according to axe-core
{code: 'color-contrast'}, // part of initial set - valid
{code: 'color-contrast'}, // <-end: Initial set of color contrast errors - valid
{code: 'aria-attributes-invalid'}, <-start of new set of errors
{code: 'color-contrast'} // <-start of another set of color contrast errors - invalid error according to axe-core
]
```

It is definitely flawed and won't work if it returns a list of **all** invalid errors but the sample output in the description is a good sample of the accuracy of the report with this hack. Since there's more time next week, I can try to spend time transitioning to axe and leaving pa11y if we decide this is too unreliable.

Reply:
This seems to work for the case described above, but it assumes that there will always be correct color contrast errors in addition to incorrect ones. if there are only invalid ones, this code will count them all as valid. I'm not sure what's best at this point, maybe this is fine because it gets us closer, but either way its probably fine